python/Quickstart.MD

```diff
@@ -142,192 +119,201 @@ Body of event request:
     "payload": {
-        "id": "fd04dd70-c162-43ef-8118-539ca01de471"
+        "id": "ITEM_ID"
     }
 }
 ```
 
 ## Response Handling
 
-The `id` in a successful response payload is the identifier of the created incident. Notice that the response is a string, not a JSON object. To get this identifier, enter the following in your Python terminal:
-```
->>> import json
->>> response_json = json.loads(result.text)
->>> incident_id = response_json.get('payload').get('id')
-```
+The `id` in a successful response payload is the identifier of the created incident. Notice that the response is a JSON object that has been converted to a string for transmission. To work with this data, convert it to a Python dictionary.  
 
-To ensure your value stored properly, enter the following in your Python terminal:
+**Type or copy the following into your Python terminal**  
 ```
->>> incident_id
-'fd04dd70-c162-43ef-8118-539ca01de471'
+response_json = json.loads(result.text)
+incident_id = response_json.get('payload').get('id')
 ```
 
+You may now reference your created incident id by using its variable name.
+
 ## Add Incident Comment
 
-Prepare a comment payload file called `comment_payload.json` containing the below:
+The APT-SCWM shared API contains processes for adding, editing, or listing comments for incidents along with other utility functions.  The header of this request body is slightly different (in this case only the `eventName` changes) and uses the same authorization configuration as the previous request.  
+If you are building or modifying your requests programmatically, you could use the `ITEM_ID` you created in the previous step.  The JSON library allows us to use `json.loads(string)` to create a dictionary from an existing JSON object and then use the `json.dumps(dictionary)` to convert our modified dictionary back to a JSON object.  
+
+**Type or copy the following into your Python terminal**  
 ```
-{
-	"payload": {
-		"processId": "4ec1229a-d46f-4fc0-878f-188d318db55a",
-		"processType": "directSupplierIncident",
-		"aptCommentBox":{
-			"aptComment":{
-				"commentText": "Please provide a list of missing items and we will replace them.",
-				"visibilityType": "Public"
-				}
-			}
-		}
+comment_request = {
+  "header": {
+    "appName": "agile-process-teams",
+    "dataspace": "default",
+    "eventName": "agile-process-teams:add-comment-for-incident:v1",
+    "headerVersion": 1,
+    "ownerId": "YOUR_OWNER_ID",
+    "processNetworkId": "YOUR_PROCESS_NETWORK_ID"
+    },
+  "payload": {
+    "processId": "my_incident_id",
+    "processType": "directSupplierIncident",
+    "aptCommentBox":{
+      "aptComment":{
+        "commentText": "Please provide a list of missing items and we will replace them.",
+        "visibilityType": "Public"
+      }
+    }
+  }
 }
 ```
+Remember that we created a dictionary object with the above data.  If we wish to update the `ITEM_ID` with the result from the previous step, we can use the `.update()` method.  
```

Comment:
Eliminate first sentence 

"To update the result, use the `ITEM_ID` and the `.update()` method."

Reply:
reworded